libwarpaffine/main.cpp

```diff
@@ -533,6 +536,8 @@ int libmain(int argc, char** _argv)
         doWarp.DoOperation();
         WaitUntilDone(app_context, doWarp);
 
+        reader->EnumerateAttachments([&writer, &reader](int index, const libCZI::AttachmentInfo& info){writer->AddAttachment(reader->ReadAttachment(index)); return true;});
```

Comment:
It's certainly nit-picking, but I am bit bothered by the fact we introduce a new variable 'reader'. It's wasting a couple of bytes, and not necessary. I'd suggest this - which maybe looks a bit more cumbersome, granted:

```suggestion
        get<0>(reader_and_stream)->EnumerateAttachments(
            [&writer, &reader_and_stream](int index, const libCZI::AttachmentInfo& info) -> bool
            {
                writer->AddAttachment(get<0>(reader_and_stream)->ReadAttachment(index));
                return true;
            });
```